src/scrab-bag.js

```diff
@@ -1,41 +1,111 @@
-'use strict'
+const TILES_TOTALS = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+  _: 2
+};
+
+const exampleOutput = {
+  10: ['E'],
+  9: ['I'],
+  8: ['A'],
+  7: ['O'],
+  5: ['R', 'N', 'T'],
+  4: ['U', 'L', 'D'],
+  3: ['G', 'S'],
+  2: ['P', 'H', 'F', 'V', 'W'],
+  1: ['B', 'Y', 'J', 'K', 'M', 'Q', 'C', 'Z', '_'],
+  0: ['X']
+}
```

Comment:
could probably delete example output? this isn't actually used, it was just for reference?